src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -39,12 +39,12 @@ export default (opts) => {
   }
 
   return {
-    aiPromptTemplateLoader: gravityLoader((id) => `a_i_prompt_template/${id}`),
-    aiPromptTemplatesLoader: gravityLoader(
-      "a_i_prompt_templates",
-      {},
-      { headers: true }
+    aiPromptTemplateLoader: gravityUncachedLoader(
+      (id) => `a_i_prompt_template/${id}`
     ),
+    aiPromptTemplatesLoader: gravityUncachedLoader("a_i_prompt_templates", {
```

Comment:
Yea if you want realtime the regular loader in these 'without' factories will always cache GET requests, unless being performed in the context of a mutation.